src/BundleAdjustmentArtifactFunctions.jl

```diff
@@ -33,6 +34,21 @@ function get_first_name_and_group(dataframe::DataFrame)
   return name, group
 end
 
+"""get_names_and_groups(dataframe::DataFrame)
+
+Return an array of tuples of the names and groups in the DataFrame
+"""
+function get_names_and_groups(dataframe::DataFrame)
+  n = length(dataframe[!, 1])
+  names_and_groups = Vector{Tuple}(undef,n)
+  for i = 1 : n
+    name = dataframe[i, :name]
+    group = dataframe[i, :group]
+    names_and_groups[i] = (name, group)
+  end
+  return names_and_groups
+end
```

Comment:
What is the advantage of this compared to extracting a DataFrame? For example, something like `dataframe[:, [:name, :group]]` (or some variation on that syntax; I didn't test)?

Reply:
The idea was to obtain an array which has elements that can be directly used as arguments of the constructor for convenience sake, but I didn't test if there was a way to do it directly from the dataframe so I will also need to test it